omero/sysadmins/server-upgrade.rst

```diff
@@ -324,6 +324,8 @@ up to date to ensure that security updates are applied:
       $ pip install --upgrade 'omero-py>=\ |version_py|'
 
 
+Since version ``5.13.0,`` the use of ``omero certificates`` is required to ensure that an OMERO server installation has, at minimum, a self-signed certificate.
```

Comment:
:+1: for adding a note in the upgrade page. Is it worth putting under a dedicated heading as it's not really related to `Dependencies`?

Two comments:
- `Since version 5.13.0` refers to OMERO.py 5.13.0 but server certificates might already have been required e.g. for Java clients
- `the use of ``omero certificates`` is required`: I would rephrase the requirement since what is needed is the presence and the configuration of a server certificate. `omero certificates` is definitely the recommended tool for generating self-signed certificates but it is also possible to deploy them manually e.g. as described in https://omero.readthedocs.io/en/stable/sysadmins/client-server-ssl.html.

Reply:
propose changes in [8e221b9](https://github.com/ome/omero-documentation/pull/2285/commits/8e221b9d1f1ed808916e7571bcba7f6274b24008)